gui/system/forms.py

```diff
@@ -2894,9 +2894,9 @@ class CertificateEditForm(ModelForm):
     def __init__(self, *args, **kwargs):
         super(CertificateEditForm, self).__init__(*args, **kwargs)
 
-        self.fields['cert_name'].widget.attrs['readonly'] = True
-        self.fields['cert_certificate'].widget.attrs['readonly'] = True
-        self.fields['cert_privatekey'].widget.attrs['readonly'] = True
+        self.fields['cert_name'].widget.attrs['readonly'] = False
+        self.fields['cert_certificate'].widget.attrs['readonly'] = False
+        self.fields['cert_privatekey'].widget.attrs['readonly'] = False
```

Comment:
Doesnt removing these lines have the same effect?

Reply:
Well yes, the effect would be the same. However, I wanted to make it clear like this in case of future changes. @william-gr

Reply:
Well, lets just remove all `__init__` method. I dont think just having a few attributes being set to False with help with that. Plus all the logic is being moved to middleware.